plugin/src/main/java/com/iridium/iridiumskyblock/listeners/BlockFormListener.java

```diff
@@ -0,0 +1,37 @@
+package com.iridium.iridiumskyblock.listeners;
+
+import com.cryptomorin.xseries.XMaterial;
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.IslandUpgrade;
+import org.bukkit.Material;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockFormEvent;
+
+import java.util.*;
+
+public class BlockFormListener implements Listener {
+
+    @EventHandler
+    public void onBlockForm(BlockFormEvent event) {
+        if (event.getNewState().getType().equals(Material.COBBLESTONE)) {
```

Comment:
Support for basalt (?) generators?

Reply:
Yep, that will replace the nether generators, ill do that later

Reply:
Isn't it possible to build them in the overworld too?

Reply:
:shrug: probabbly